### PR TITLE
CET-9697 Enforce semicolons

### PR DESCRIPTION
**Pull request by dusan-kovacik-profiq**
## Change description
https://cortex1.atlassian.net/browse/CET-9697
This PR adds an ESLint rule to enforce semicolons.

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [X] Improvement (improves codebase without changing functionality)

### Development

- [X] The changelog has been updated as appropriate
- [X] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: src/utils/dates.ts [hunk 1]
```diff
@@ -20,4 +20,4 @@ export const daysUntil = (expirationDate: string | null) => {
   return !isNil(expirationDate)
     ? moment(expirationDate).diff(moment(), 'days')
     : undefined;
-}
```
Comment (PavelZurek) [inline src/utils/dates.ts:23]: We should also add rule for an empty row at the end of file. This is not related to this PR.

**Review by PavelZurek — DISMISSED** [review]
Looks good.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.11.7
```
Comment (FoodProduct) [inline CHANGELOG.md:3]: Since this is not a functional change, we shouldn't bump the package version

**Review by FoodProduct — COMMENTED** [review]
Thank you for adding the rule! However, let's avoid bumping the package version

File: CHANGELOG.md (hunk 2 above)
Comment (dusan-kovacik-profiq) [reply CHANGELOG.md:3]: Ok, that makes sense. And should I add a note about adding the rule to the changelog (to the current version) or we don't need the note there?

Comment (FoodProduct) [reply CHANGELOG.md:3]: No need for a note in the changelog. The changelog is intended for consumers of the library.

**Review by FoodProduct — APPROVED** [review]

File: src/utils/dates.ts (hunk 1 above)
Comment (cristinabuenahora) [reply src/utils/dates.ts:23]: @dusan-kovacik-profiq can you add this next?

Comment (dusan-kovacik-profiq) [reply src/utils/dates.ts:23]: Ok, as you have merged it, I will create a new one :)
